src/components/Fetch/index.js

```diff
@@ -514,6 +514,11 @@ class Fetch extends React.Component {
     event.preventDefault();
   }
 
+  handleChangeFind(caseSensitive) {
+    this.find(caseSensitive);
+    this.showFind();
```

Comment:
showFind is the function that renders next/prev buttons. This call does not appear to be required